dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/view/Page.java

```diff
@@ -51,29 +51,20 @@
 @ToString
 @EqualsAndHashCode
 public class Page<T> {
-  @OpenApi.Property(value = OpenApi.EntityType[].class)
+
+  @JsonIgnore private final String key;
+
+  /** Is serialized under the dynamic {@link #key} using {@link #getDynamicItems}. */
   @JsonIgnore
-  @JsonAnyGetter
-  private final Map<String, List<T>> items = new LinkedHashMap<>();
+  @OpenApi.Property(value = OpenApi.EntityType[].class)
```

Comment:
If you wanted to make this work with the annotation on the classes just be aware that (I think) you can also do this per method. The class level is just the default for the methods in the controller. Sometimes defining what the actual `EntityType` is makes more sense per method as not most methods in a controller would be about that type.